narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -365,13 +397,13 @@ function process_zip_list_file {
 
 		check_if_tag_exists liferay-dxp "${tag_name_new}" && continue
 
-		check_ignore_zip_file "${hotfix_zip_file}" "${release_version}" && continue
+		check_ignore_zip_file "${hotfix_zip_file}" && continue
```

Comment:
Why did you remove $release_version?

Reply:
It was before we talked about putting there arguments that the code don't need. Should I put it back?